ember-app/app/controllers/issue.js

```diff
@@ -1,7 +1,15 @@
 import Ember from 'ember';
+import IssueEvent from "app/mixins/events/issue";
+import Messaging from "app/mixins/messaging";
 
-var IssueController = Ember.Controller.extend({
+var IssueController = Ember.Controller.extend(
+  IssueEvent, Messaging, {
   needs: ["application"],
+
+  //Subscribes to Messages in the Route after the model is set
+  subscribeDisabled: true,
```

Comment:
could we just unsub/sub when the observer fires?

basically the same thing, ie; first pass would sub to `undefined issue.undefined.event` then when the observer fires it unsubs `undefined issue.undefined.event` and subs the bound values

probably going to be nessasary to make the issue controller work in the current state. 

my guess is there is a bug now that leaves old subscriptions when you change issues